### DISCUSSION TITLE
UI/Build versions - Preview vs non-Preview tabs

### DISCUSSION THREAD

**Discussion by nerdCopter** [supermerill/SuperSlicer#1258] [Q&A] [answered]
I'm not 100% sure which alphas i should be downloading as there are two differing UI's.
fwiw, i prefer the one with slice & gcode tabs.  

Can someone explain? for example `2.3.56.1` and commit `896006a` have them, but `2.3.56.2` and commit `d2dc7c3` do not.

![2021-06-05_08-19](https://user-images.githubusercontent.com/56646290/120893065-cbee5700-c5d6-11eb-8a8e-c2389af01711.png)

**Comment by supermerill** [comment, marked as answer]
I think you're on linux.
I changed the default to the old one because some distribution are a bit buggy with the new one.
Change it in preferences -> gui (at the bottom)
